src/bundler.js

```diff
@@ -322,7 +328,17 @@ function processJsBundle(options, jsBundle, bundleDir, jsFiles, bundleName, cb)
 
                 readTextFile(filePath, function(code) {
 
-                    var compileOptions = getProcessCodeOptions(code, undefined, filePath, jsPathOutput, bundleDir, bundleStatsCollector, options);

```

Comment:
This `getProcessCodeOptions` helper wasn't really helping and it was taking in a million unnamed parameters.